accounts-db/src/accounts.rs

```diff
@@ -44,12 +45,47 @@ use {
     },
 };
 
+// max number of accts in a default size entry 64 * 256
+const BATCH_ACCOUNTS_LOOKUP_TABLE_SIZE: usize = 16384;
```

Comment:
I used 256 because I read somewhere like [here](https://solana.com/docs/advanced/lookup-tables) that ALT will enable 256 accounts per tx.

It might be that code and documentation are not aligned on this point.

Reply:
Yeah docs could definitely be better!

256 is the maximum number of accounts **possible** with our current transaction format, because transaction indexes are represented by a `u8`.
But we check number of account locks before processing, so anything that gets to locking should have fewer than `64` (and once `9LZdXeKGeBV6hRLdxS1rHbHoEUsKqesCC2ZAPTPKJAbK` is activated `128`).